packages/near-fast-auth-signer/src/components/SignMultichain/SignMultichain.tsx

```diff
@@ -267,7 +245,7 @@ function SignMultichain() {
           size="large"
           label={inFlight ? 'Loading...' : 'Approve'}
           onClick={onConfirm}
-          disabled={inFlight || !isValid || typeof amountInfo.price !== 'number'}
+          disabled={inFlight || !isValid || !firebaseUserLoading || !firebaseUser || typeof amountInfo.price !== 'number'}
```

Comment:
`!firebaseUserLoading` -> `firebaseUserLoading` need to be changed

Reply:
True, updated :)